src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,126 @@
+package duke.util;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Prints output messages from Duke.
+ *
+ * @author Jicson Toh
+ */
+public class Ui {
+
+    /**
+     * Prints welcome message.
+     */
+    public void showWelcome() {
+        System.out.println("Hello! I'm Botson\n"
+                + "What can I help you with? :-)\n"
+                + "--------------------------");
+    }
+
+    /**
+     * Prints exit message.
+     */
+    public void exit() {
+        System.out.println("Goodbye! Hope to see you again soon!\n"
+                + "--------------------------");
+    }
+
+    /**
+     * Prints no such task error.
+     */
+    public void noSuchTaskError() {
+        System.out.println("OOPS!!! Error: No Such Task :-(\n"
+                + "--------------------------");
+    }
+
+    /**
+     * Prints unknown error message.
+     */
+    public void showError() {
+        System.out.println("An error occurred.");
+        showLine();
+    }
+
+    /**
+     * Prints error message.
+     */
+    public void showError(String error) {
+        System.out.println(error);
+        showLine();
+    }
+
+    /**
+     * Prints cannot understand error message.
+     */
+    public void cannotUnderstandError() {
+        System.out.println("OOPS!!! I'm sorry, but I don't know what that means :-(\n"
+                + "--------------------------");
+    }
+
+    /**
+     * Prints the removed task from the list and updates the list count.
+     * @param removedTask task to be removed.
+     * @param size size of list after task is remove.
+     */
+    public void removeTask(String removedTask, int size) {
```

Comment:
```suggestion
    public void showRemovedTask(String removedTask, int size) {
```
Since this method is used by the Ui object, perhaps it would make more sense by naming it to indicate that its printing out a message rather than actually removing a task? Similarly for the below task related methods

Reply:
agreed, will take note thanks!